j/jackson-core/jackson-core_ubi_9.3.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package :jackson-core
+# Version : 2.17.0
+# Source repo : https://github.com/FasterXML/jackson-core/tree/2.17
+# Tested on : UBI 9.3
+# Language : Java
+# Travis-Check : True
+# Script License: Apache License, Version 2 or later
+# Maintainer : Pratibh Goshi<pratibh.goshi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ========== platform using the mentioned version of the package.
+# It may not work as expected with newer versions of the
+# package and/or distribution. In such case, please
+# contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_NAME=jackson-core
+PACKAGE_VERSION=jackson-core-2.17.0
```

Comment:
As suggested by @nikhil-kalbande on your other PR, please update the package version here as well.
PACKAGE_VERSION=${1:-jackson-core-2.17.0}.